lib/prompt.ts

```diff
@@ -252,8 +252,10 @@ const metadataSystemPrompt = `You are an AI assistant tasked with evaluating the
 Analyze the extraction response and determine if the task is completed or if more information is needed.
 
 Strictly abide by the following criteria:
-1. If you are certain that the instruction is completed, set the completion status to true, even if there are still chunks left.
-2. If there could still be more information to extract and there are still chunks left, set the completion status to false.`;
+1. If the instruction has been satisfied by the current extraction response, ALWAYS set completion status to true and stop processing, regardless of remaining chunks.
```

Comment:
Nit: I think wording here could be "Once the instruction has been satisfied..." instead of "if". It might help in situations where you have to scroll a little and stop for example

Reply:
Valid—updated!